apps/win32/package.json

```diff
@@ -63,20 +63,22 @@
     "typescript": "3.8.3",
     "webdriverio": "5.22.4"
   },
-  "peerDependencies": {
-    "react": "16.13.1",
-    "react-native": "^0.63.4"
-  },
   "jest": {
     "preset": "react-native"
   },
   "rnx-kit": {
     "reactNativeVersion": "0.63",
+    "kitType": "app",
     "bundle": {
       "targets": [
         "win32"
       ],
       "entryPath": "index.js"
-    }
+    },
+    "capabilities": [
+      "core-android",
+      "react",
+      "svg"
+    ]
```

Comment:
why does the win32 test app have android capabilities?

Reply:
It's because it has a dependency on `react-native`. We don't have a good way of saying "I just want core". So I picked `core-android`. Will clean this up in a separate PR.